uber/templates/groups/form.html

```diff
@@ -25,147 +25,194 @@
+                        (The email subject will be "Your {{ c.EVENT_NAME }} Dealer registration has been [waitlisted | declined]")
                         <textarea name="email", rows="5" cols="50"></textarea> <br/>
                         <button onClick="unapprove('waitlisted'); return false;">Waitlist</button>
                         <button onClick="unapprove('declined'); return false;">Reject</button>
                     </div>
                 {% endif %}
             {% endif %}
-        </td>
-    </tr>
-    <tr>
-        <td valign="top"> <b>What do they sell?</b> </td>
-        <td> <textarea rows="3" cols="30" name="wares">{{ group.wares }}</textarea>
-    </tr>
-    <tr>
-        <td> <b>Tax ID</b> </td>
-        <td> <input type="text" name="website" value="{{ group.website }}" />
-    </tr>
-    <tr>
-        <td valign="top"> <b>Description</b> </td>
-        <td> <textarea rows="3" cols="30" name="description">{{ group.description }}</textarea>
-    </tr>
-    <tr>
-        <td valign="top"> <b>Special Requests</b> </td>
-        <td> <textarea rows="3" cols="30" name="special_needs">{{ group.special_needs }}</textarea>
-    </tr>
-    <tr>
-        <td valign="top"> <b>Address</b> </td>
-        <td> <textarea rows="3" cols="30" name="address">{{ group.address }}</textarea>
-    </tr>
+        </div>
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-2 control-label">What do they sell?</label>
+        <div class="col-sm-6">
+            <textarea rows="3" cols="30" name="wares">{{ group.wares }}</textarea>
+        </div>
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-2 control-label">Tax ID</label>
+        <div class="col-sm-6">
+            <input type="text" name="website" value="{{ group.website }}" />
+        </div>
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-2 control-label">Description</label>
+        <div class="col-sm-6">
+            <textarea rows="3" cols="30" name="description">{{ group.description }}</textarea>
+        </div>
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-2 control-label">Special Requests</label>
+        <div class="col-sm-6">
+            <textarea rows="3" cols="30" name="special_needs">{{ group.special_needs }}</textarea>
+        </div>
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-2 control-label">Address</label>
+        <div class="col-sm-6">
+            <textarea rows="3" cols="30" name="address">{{ group.address }}</textarea>
+        </div>
+    </div>
 {% endif %}
-<tr>
-    <td valign="top"> <nobr>Admin Notes:</nobr> </td>
-    <td> <textarea name="admin_notes" rows="3" style="width:66%">{{ group.admin_notes }}</textarea> </td>
-</tr>
-</table>
-
-<br/>
-
-<div style="margin-left:25%">
-    <input type="submit" value="Upload Info" />
-    &nbsp;&nbsp;&nbsp;&nbsp;
-    <a href="index">Return to Group List</a>
+
+{% include "groupextra.html" %}
```

Comment:
Oh, I see - I would think groupextra.html would be a user-facing page, much like regextra.html is. I supposed regextra.html is used on both the admin side and the user side.

Come to think of it, there's no way to edit a group's information from the user side after you create it, is there? The current group management page just lets you control members of the group.

We probably want to add that in at some point. Should we also have specialized "regextraadmin" and "groupextraadmin" forms so a plugin can insert admin-only forms? Or would that be better-solved with some sort of check inside the extra forms (maybe the PAGE_PATH check) to enable certain fields to be admin-only?